frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.unit.spec.tsx

```diff
@@ -0,0 +1,119 @@
+import { renderWithProviders, screen } from "__support__/ui";
+import type { EmbeddingEntityType } from "metabase/embedding-sdk/store";
+import * as domUtils from "metabase/lib/dom";
+import {
+  createMockEmbedOptions,
+  createMockEmbedState,
+  createMockState,
+} from "metabase-types/store/mocks";
+
+import { BrowseNavSection } from "./BrowseNavSection";
+
+describe("BrowseNavSection", () => {
+  afterEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  it("should render Models, Databases, and Metrics when not embedding in iframe", () => {
+    setup();
+
+    expect(
+      screen.getByRole("listitem", { name: "Browse models" }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole("listitem", { name: "Browse databases" }),
+    ).toBeInTheDocument();
+    expect(
+      screen.getByRole("listitem", { name: "Browse metrics" }),
+    ).toBeInTheDocument();
+  });
+
+  describe("interactive embedding with `entity_types` (EMB-229)", () => {
```

Comment:
have the same amount of test cases as in https://github.com/metabase/metabase/pull/55331/files#diff-349612a9495576c767b8c57e8a90da1456178541a172e026df977051b14ae115R167